### PR TITLE
#119 refactor main window into component

### PR DESCRIPTION
**Pull request by bejow**
## Related Ticket/Issue
#119 

## Description
Adding a Layout Component to reduce code repetion

### CODE REVIEW

File: src/functional_components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -16,7 +17,7 @@ const LandingPage = ({ webId, history }) => {
     console.log(webId);
 
     return (
-        <div className={styles.grid}>
+        <Layout className={styles.grid} hideToolbar>
```
Comment (ludwigschubi) [inline src/functional_components/LandingPage/LandingPage.js]: same here

File: src/functional_components/ToolbarButtons/ToolbarButtons.module.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .toolbarIcon {
     cursor: pointer;
-    margin: 0 20px 0 20px;
+    margin-left: 40px;
```
Comment (ludwigschubi) [inline src/functional_components/ToolbarButtons/ToolbarButtons.module.css]: Even though it isn't necessary you might still want to remove the margin for the first (from left) button

File: src/stateful_components/Drive/Drive.js [hunk 3]
```diff
@@ -321,6 +322,21 @@ class Drive extends React.Component {
         // localStorage.setItem('appState', JSON.stringify(this.state));
     }
 
+    toolbarRight() {
```
Comment (ludwigschubi) [inline src/stateful_components/Drive/Drive.js]: Why does this have to be a function and not an object?

**Comment by ludwigschubi** [comment]
@bejow Just some tiny things

File: src/functional_components/LandingPage/LandingPage.js (hunk 1 above)
Comment (bejow) [reply src/functional_components/LandingPage/LandingPage.js]: Same what?

File: src/stateful_components/Drive/Drive.js [hunk 4]
```diff
@@ -47,6 +48,18 @@ class Drive extends React.Component {
             renamedItem: undefined,
         };
 
+        this.toolbarRight = (
```
Comment (ludwigschubi) [inline src/stateful_components/Drive/Drive.js]: Is there a specific reason why toolbarRight is declared in the constructor as a property and toolbarLeft is declared as a variable before render?

**Comment by ludwigschubi** [comment]
Otherwise looking good

**Pull request merged**